plugins/tracking/hooks.js

```diff
@@ -39,6 +51,19 @@ module.exports = {
 
   pluginsConfigured() {
     DOMUtils.appendScript(document.head, segmentScript);
+
+    window.analytics.ready(function () {
+      function updateTrackJSConfiguration() {
```

Comment:
This updates the track js configuration to send the DCOS version number with every tracked error.